### ISSUE TITLE
Pongbot does not run on linux mint 19

### ISSUE THREAD

**Issue by descholar-ceo** [slack-ruby/slack-ruby-bot#270]
When I setup my new project of slack-bot, it don't start where I call `SLACK_API_TOKEN=apit_token bundle exec ruby lib/neza_sbot.rb` => neza_sbot is the same as `pongbot.rb`

So this is the log I am getting:

```ruby
Traceback (most recent call last):
        9: from lib/neza_sbot.rb:1:in `<main>'
        8: from lib/neza_sbot.rb:1:in `require'
        7: from /home/descholar/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/slack-ruby-bot-0.16.0/lib/slack-ruby-bot.rb:3:in `<top (required)>'
        6: from /home/descholar/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/slack-ruby-bot-0.16.0/lib/slack-ruby-bot.rb:3:in `require'
        5: from /home/descholar/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/slack-ruby-bot-0.16.0/lib/config/environment.rb:5:in `<top (required)>'
        4: from /home/descholar/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/slack-ruby-bot-0.16.0/lib/config/environment.rb:5:in `require'
        3: from /home/descholar/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/slack-ruby-bot-0.16.0/lib/config/application.rb:6:in `<top (required)>'
        2: from /home/descholar/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/slack-ruby-bot-0.16.0/lib/config/application.rb:6:in `require'
        1: from /home/descholar/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/slack-ruby-bot-0.16.0/lib/config/boot.rb:5:in `<top (required)>'
/home/descholar/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/slack-ruby-bot-0.16.0/lib/config/boot.rb:5:in `require': cannot load such file -- active_support (LoadError)
```

I have tried to research on the internet since Friday evening, but couldn't get any medic to my problem. My system is Linux mint 19.3 Tricia. I think it's a bug.

**Comment by dblock** [comment]
We removed a dependency on activesupport in https://github.com/slack-ruby/slack-ruby-client/pull/325, but looks like it's needed by slack-ruby-bot. I fixed that [here](https://github.com/slack-ruby/slack-ruby-bot/pull/271).

As a workaround, add `activesupport` into your Gemfile.

**Comment by dblock** [comment]
Also running pongbot I'm seeing a side-effect fixed by https://github.com/slack-ruby/slack-ruby-client/pull/336.

We need some kind of integration test here ...

**Comment by descholar-ceo** [comment]
> We removed a dependency on activesupport in [slack-ruby/slack-ruby-client#325](https://github.com/slack-ruby/slack-ruby-client/pull/325), but looks like it's needed by slack-ruby-bot. I fixed that [here](https://github.com/slack-ruby/slack-ruby-bot/pull/271).
> 
> As a workaround, add `activesupport` into your Gemfile.

Yes Thank you @dblock That worked.

**Comment by descholar-ceo** [comment]
> Also running pongbot I'm seeing a side-effect fixed by [slack-ruby/slack-ruby-client#336](https://github.com/slack-ruby/slack-ruby-client/pull/336).
> 
> We need some kind of integration test here ...

Yes sure, running it, it's showing a lot of warnings, hope that when [this PR](https://github.com/slack-ruby/slack-ruby-client/pull/336) gets merged for the next release, things will be okay
